### PR TITLE
feat: do not validate proposal if releaseNotes falsy

### PR DESCRIPTION
**Pull request by mhofman**

### CODE REVIEW

**Review by gibson042 — APPROVED** [review]
Approved, with suggestions to consider.

File: packages/synthetic-chain/public/upgrade-test-scripts/run_prepare.sh [hunk 1]
```diff
@@ -54,9 +54,13 @@ else
   echo "Upgrade info is not valid JSON: $info"
   exit $status
 fi
+validate_flag=""
+if [ "$VALIDATE_PROPOSAL" = "false" ]; then
+  validate_flag="--no-validate"
+fi
```
[the remark this answers is not in the document]
Comment (mhofman) [reply packages/synthetic-chain/public/upgrade-test-scripts/run_prepare.sh:60]: I'm rusty on my bash, what is the behavior if `SKIP_PROPOSAL_VALIDATION=false` ?

[the remark this answers is not in the document]
Comment (mhofman) [reply packages/synthetic-chain/public/upgrade-test-scripts/run_prepare.sh:60]: Yeah as I thought it doesn't support it. testing for empty-ness when plumb in a 'true' value is a footgun

**Comment by mhofman** [comment]
I have verified that this latest iteration works when applied as a patch in agoric-sdk. Once CI passes here and I get a 👍 from @turadg I will publish version 0.0.10

**Review by turadg — APPROVED** [review]

**Pull request merged**